netbox/users/models/tokens.py

```diff
@@ -180,6 +185,26 @@ def token(self, value):
                 self.key = self.key or self.generate_key()
                 self.update_digest()
 
+    @property
+    def is_expired(self):
+        """
+        Check whether the token has expired.
+        """
+        if self.expires is None or timezone.now() < self.expires:
+            return False
+        return True
+
+    @property
+    def is_active(self):
+        """
+        Check whether the token is active (enabled and not expired).
+        """
+        if not self.enabled:
+            return False
+        if self.is_expired:
+            return False
+        return True
```

Comment:
We can simplify this a bit.
```suggestion
        return self.enabled and not self.is_expired
```

Reply:
💯 
This is way better! Thank you!